learningresources/models.py

```diff
@@ -46,10 +46,6 @@ class LearningResource(models.Model):
     xa_avg_grade = models.FloatField(default=0)
     xa_histogram_grade = models.FloatField(default=0)
 
-    class Meta:
```

Comment:
Yes, it was intentional because things were breaking without it. I could have fixed it with a migration (and did, for a while), but since #80 is out there pending, I figured I'd just wait for it to be merged instead of fixing it twice.